src/cascadia/TerminalSettingsEditor/Launch.xaml

```diff
@@ -201,6 +201,17 @@
                                         Value="{x:Bind State.Settings.GlobalSettings.InitialRows, Mode=TwoWay}" />
                     </Grid>
                 </local:SettingContainer>
+
+                <!--  ConsoleHost startup mode  -->
+                <local:SettingContainer x:Name="Globals_ConsoleHostStartupMode"
+                                        x:Uid="Globals_ConsoleHostStartupMode"
+                                        Style="{StaticResource ExpanderSettingContainerStyle}">
+                    <muxc:RadioButtons AutomationProperties.AccessibilityView="Content"

```

Comment:
PR #12833 replaced all of our RadioButtons with ComboBoxes. Make sure you update this one too please